### ISSUE TITLE
[BUG] SDK doesn’t follow `priority` config for failover

### ISSUE THREAD

**Issue by kysely** [superfaceai/one-sdk-js#130]
Given the following super.json, `plivo` was configured before `twilio`, but the priority sets `twilio` as primary.

```json
{
  "profiles": {
    "communication/send-sms": {
      "version": "1.0.1",
      "defaults": {
        "SendMessage": {
          "providerFailover": true
        }
      },
      "priority": [
        "twilio",
        "plivo"
      ],
      "providers": {
        "plivo": {
          "defaults": {
            "SendMessage": {
              "retryPolicy": "circuit-breaker",
              "input": {
                "PLIVO_AUTH_ID": "invalid"
              }
            }
          }
        },
        "twilio": {
          "defaults": {
            "SendMessage": {
              "retryPolicy": "circuit-breaker",
              "input": {
                "TWILIO_SID": "$TWILIO_USERNAME"
              }
            }
          }
        }
      }
    }
  },
  "providers": {
    "plivo": {
      "security": [{
        "id": "basic",
        "username": "$PLIVO_USERNAME",
        "password": "$PLIVO_PASSWORD"
      }]
    },
    "twilio": {
      "security": [{
        "id": "basic",
        "username": "$TWILIO_USERNAME",
        "password": "$TWILIO_PASSWORD"
      }]
    }
  }
}
```

## Expected Behavior

OneSDK should start the use case perform with `twilio` & fail over to `plivo`. It should report `twilio` as the primary provider.

## Current Behavior

OneSDK starts the use case perform with `plivo` & reports it as the primary for insights.

## Steps to Reproduce

1. install https://superface.ai/communication/send-sms
2. install `plivo`, then `twilio`
3. use the `super.json` from above
4. run

**Comment by lukas-valenta** [comment]
Fixed by #137

**Comment by kysely** [comment]
Confirming that it works with [v0.0.36](https://github.com/superfaceai/one-sdk-js/releases/tag/v0.0.36). Thanks.
